### ISSUE TITLE
[Bug] CardBrowser: findViewById(R.id.toolbar_spinner) must not be null

### ISSUE THREAD

**Issue by david-allison** [ankidroid/Anki-Android#13698]
https://ankidroid.org/acra/app/1/bug/46327/report/0a46f363-230b-4978-ad18-6e45dd686f02

```
java.lang.NullPointerException: findViewById(R.id.toolbar_spinner) must not be null
	at com.ichi2.anki.CardBrowser.onCollectionLoaded(CardBrowser.kt:211)
	at com.ichi2.anki.AnkiActivity.startLoadingCollection$lambda$0(AnkiActivity.kt:18)
	at com.ichi2.anki.AnkiActivity.i(AnkiActivity.kt:1)
	at com.ichi2.anki.u0.execute(R8$$SyntheticClass:3)
	at com.ichi2.async.CollectionLoader$load$1.invokeSuspend(CollectionLoader.kt:68)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:12)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:119)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7891)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@92b0c26, Dispatchers.Main.immediate]
```

* 2.16alpha97
* non-rust Backend
* Google Pixel 4 on grapheneos

<details><summary>Logcat</summary>

```

--------- beginning of main
04-24 16:20:01.478 I/AnkiDroid(27323): initialize()
04-24 16:20:01.737 I/AnkiDroid(27323): Not participating in analytics sample (sample percentage vs random: 10 46)
04-24 16:20:01.737 I/AnkiDroid(27323): setOptIn(): from false to false
04-24 16:20:01.737 I/AnkiDroid(27323): Not participating in analytics sample (sample percentage vs random: 10 49)
04-24 16:20:01.740 I/AnkiDroid(27323): Creating notification channel with id/name: General Notifications/AnkiDroid
04-24 16:20:01.741 I/AnkiDroid(27323): Creating notification channel with id/name: Synchronization/Synchronisation
04-24 16:20:01.742 I/AnkiDroid(27323): Creating notification channel with id/name: Global Reminders/Cartes à réviser
04-24 16:20:01.742 I/AnkiDroid(27323): Creating notification channel with id/name: Deck Reminders/Rappels
04-24 16:20:01.743 I/AnkiDroid(27323): Creating notification channel with id/name: Scoped Storage/Transfert du stockage
04-24 16:20:01.751 I/AnkiDroid(27323): AnkiDroidApp: Starting Services
04-24 16:20:01.752 W/AnkiDroid(27323): AnkiDroidApp/ Boot Service did not execute - no permissions
04-24 16:20:01.770 I/AnkiDroid(27323): AnkiActivity::onCreate - CardBrowser
04-24 16:20:01.771 I/AnkiDroid(27323): Setting theme to DARK
04-24 16:20:01.918 I/AnkiDroid(27323): AnkiActivity::onStart - CardBrowser
04-24 16:20:01.932 I/AnkiDroid(27323): AnkiActivity::onResume - CardBrowser
04-24 16:20:02.216 I/AnkiDroid(27323): (Re)opening Database: /storage/emulated/0/AnkiDroid/collection.anki2
04-24 16:20:02.372 I/AnkiDroid(27323): Opening Collection Log
04-24 16:20:02.391 I/AnkiDroid(27323): CollectionLoader obtained collection
```
</details>

**Comment by soCallmeAdityaKumar** [comment]
please Assign it to me

**Comment by lukstbit** [comment]
@soCallmeAdityaKumar You are now assigned, go for it.

**Comment by SanjaySargam** [comment]
@soCallmeAdityaKumar Are you still working on it?

**Comment by soCallmeAdityaKumar** [comment]
> @soCallmeAdityaKumar Are you still working on it?

Yes

**Comment by dae** [comment]
Tentatively assuming this is fixed; please comment if not fixed in a 2.17 alpha, and it can be re-opened.
